backstory/pom.xml

```diff
@@ -4,7 +4,7 @@
   <modelVersion>4.0.0</modelVersion>
 
   <groupId>com.google.sps</groupId>
-  <artifactId>portfolio</artifactId>
+  <artifactId>backstory</artifactId>
```

Comment:
FYI: for future reference, this could have been a PR on its own (would have landed quicker).

Reply:
Good to know. Thank you.